Client/Frontend/Menu/Items/PagingMenuItemCollectionViewLayout.swift

```diff
@@ -0,0 +1,76 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+class PagingMenuItemCollectionViewLayout: UICollectionViewLayout {
+
+    var maxNumberOfItemsPerPageRow: Int = 0
+    var lineSpacing: CGFloat = 0
+    var interitemSpacing: CGFloat = 0
+    var cellSize: CGSize = CGSize.zero
```

Comment:
I had no idea CGSize had a size property! And all this time I've been using CGSizeZero...